playbook-control-room-glusterfs.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: control_room_all
+  remote_user: sirius
+  become: true
+  become_method: sudo
+  become_user: root
+
+  pre_tasks:
+    - name: Include distribution-dependent variables
+      include_vars: "{{ item }}"
+      vars:
+        possible_var_files:
+          - "group_vars/{{ ansible_distribution }}-{{ ansible_distribution_release }}.yml"
+          - "group_vars/{{ ansible_distribution }}.yml"
+          - "group_vars/{{ ansible_os_family }}.yml"
+      loop: "{{ q('first_found', possible_var_files, errors='ignore') }}"
+
+  roles:
+    - role: lnls-ans-role-glusterfs
```

Comment:
Should we also include glusterfs role into the playbook `playbook-desktops.yml`?

Reply:
This will be a tricky one, the playbook-desktops.yml will target more than just the control room.  There I plan to use the `group_namas` variable to check if the role should go or not https://docs.ansible.com/ansible/latest/reference_appendices/special_variables.html. It seems to be a good fix.